### ISSUE TITLE
[bug]: Verification fails for 0.16.4 and 0.16.3

### ISSUE THREAD

**Issue by schildbach** [lightningnetwork/lnd#7809]
```
> docker run --rm --entrypoint="" lightninglabs/lnd:v0.16.4-beta /verify-install.sh v0.16.4-beta
[…]
ERROR: Not enough valid signatures found!
  Valid signatures found: 0
  Valid signatures required: 5

  Make sure the release v0.16.4-beta contains the required 
  number of signatures on the manifest, or wait until more 
  signatures have been added to the release.
```

**Comment by schildbach** [comment]
This issue could be fixed by merging https://github.com/lightningnetwork/lnd/pull/7758

**Comment by setpill** [comment]
> This issue could be fixed by merging #7758

Only partially - there's only 3 signatures available for 0.16.4 currently.

**Comment by ellemouton** [comment]
yep - more sigs coming soon
